ext/mysqli/tests/bug81335.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
+Bug #81335: Packets out of order after connection timeout
+--EXTENSIONS--
+mysqli
+--SKIPIF--
+<?php
+if (getenv("SKIP_SLOW_TESTS")) die("skip slow test");
+
+if (!defined('MYSQLI_STORE_RESULT_COPY_DATA')) die('skip requires mysqlnd');
```

Comment:
Please don't forget to remove this line. See https://github.com/php/php-src/commit/6fb1c7a7427582641ab25c128b871fbcaa28dfe0 and https://wiki.php.net/rfc/mysqli_support_for_libmysql

Since the lowest possible target branch is 8.2 and the support for libmysql was removed in 8.2 we don't need to worry about this check at all. You can safely remove it even on 8.2.

Reply:
applied